spinn_front_end_common/interface/spinnaker.cfg

```diff
@@ -228,7 +228,7 @@ n_samples_per_recording_entry = 100
 turn_off_board_after_discovery = False
 
 [Java]
-use_java = False
+use_java = True
```

Comment:
NO!
We do NOT want Java to be on by Default!

Reply:
done. no idea how that slipped